pkg/events/events.go

```diff
@@ -80,6 +80,17 @@ type Event struct {
 	Params       []trace.ArgMeta
 }
 
+func (e *Event) IsASignatureEvent() bool {
+
+	for _, s := range e.Sets {
+		if s == "signatures" {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
This isn't yet being used. Will it be checked in a hot path? If so, could it be a preset boolean flag?

Reply:
@geyslan This is used to avoid the argument name validation when booting tracee, for example, if you do `-t security_open_file.args.pathname!=X`, there is a validation to check if the `pathaname` is a paramenter declared on the event when tracee is starting. 

As signature events the parameters are dynamic, we are skipping the validation so we don't fail the boot: https://github.com/aquasecurity/tracee/pull/2713/files#diff-a15ccb695c12a7219d07ca60dd20a3e26df991cb65d2313dcd593b5b5295a55cR104